sql/src/main/java/io/crate/execution/engine/collect/stats/FilteredLogSink.java

```diff
@@ -22,36 +22,47 @@
 
 package io.crate.execution.engine.collect.stats;
 
-import io.crate.data.Input;
-import io.crate.execution.engine.collect.NestableCollectExpression;
-import io.crate.expression.InputCondition;
+import io.crate.expression.ExpressionsInput;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.message.Message;
+import org.elasticsearch.common.logging.Loggers;
 
 import javax.annotation.Nonnull;
 import java.util.Iterator;
-import java.util.List;
+import java.util.function.Function;
 
 public final class FilteredLogSink<T> implements LogSink<T> {
 
-    private final Input<Boolean> filter;
-    private final List<NestableCollectExpression<T, ?>> filterExpressions;
+    /**
+     * This logger name is documented in the docs for the {@link JobsLogService#STATS_JOBS_LOG_PERSIST_FILTER} setting.
+     * Take care when changing the class name or package to update the docs.
+     */
+    private static final Logger LOGGER = Loggers.getLogger(FilteredLogSink.class);
```

Comment:
should we maybe rename this to e.g. `STATEMENT_LOGGER` and fix the logger name/prefix to highlight the use case of this logger?

Reply:
Good idea, any suggestions for the name/prefix?

Reply:
hm not really, `query stats`?

Reply:
Named it `StatementLog`